include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -2917,7 +2922,7 @@ hipError_t hipLaunchCooperativeKernel(const void* f, dim3 gridDim, dim3 blockDim
  * @param [in] numDevices               Size of the launchParamsList array.
  * @param [in] flags                    Flags to control launch behavior.
  *
- * @returns hipSuccess, hipInvalidDevice, hipErrorNotInitialized, hipErrorInvalidValue
+ * @returns hipSuccess, hipInvalidDevice, hipErrorNotInitialized, hipErrorInvalidValue, hipErrorCooperativeLaunchTooLarge
```

Comment:
Where are the corresponding changes in `hipLaunchCooperativeKernelMultiDevice` and `hipLaunchCooperativeKernel` functions implementation, which handle erroneous situations with emitting `hipErrorCooperativeLaunchTooLarge` error code?

Reply:
The corresponding changes in cooperative launch APIs are coming in a separate PR next. The purpose of this PR is to just add the missing flags in the headers.